src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -266,7 +266,10 @@ public void afterNavigateTo(String url, WebDriver driver) {
       User user = method.getAnnotation(Execute.class).asUser();
       if (user == User.ANONYMOUS) {
```

Comment:
the logic results in, for Anonymous annotation do nothing, for any other case log in the user